<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteDialog</class>
 <widget class="QDialog" name="DeleteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>271</width>
     <height>301</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>张三</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/image/zhangsan.png</normaloff>:/image/zhangsan.png</iconset>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>李四</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/image/lisi.png</normaloff>:/image/lisi.png</iconset>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>王五</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/image/wangwu.png</normaloff>:/image/wangwu.png</iconset>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>赵六</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/image/zhaoliu.png</normaloff>:/image/zhaoliu.png</iconset>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>孙七</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/image/sunqi.png</normaloff>:/image/sunqi.png</iconset>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>周八</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/image/zhouba.png</normaloff>:/image/zhouba.png</iconset>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>吴九</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/image/wujiu.png</normaloff>:/image/wujiu.png</iconset>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>郑十</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/image/zhengshi.png</normaloff>:/image/zhengshi.png</iconset>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="DeleteButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DisPlayButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>显示</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ReturnButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/background_meitu_2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>listWidget</zorder>
  <zorder>DeleteButton</zorder>
  <zorder>DisPlayButton</zorder>
  <zorder>ReturnButton</zorder>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
